public/javascripts/landing.js

```diff
@@ -0,0 +1,13 @@
+
+function openForms(e) {
+    e.preventDefault();
+    formName = e.currentTarget.name;
+    tabForm(e, formName);
+    document.getElementById("loginForm").style.height = "100%";
+}
+
+function closeForms() {
+    document.getElementById("loginForm").style.height = "0%";
```

Comment:
An alternative to manually changing the `.style.height` would be to toggle on/off a certain class, like adding `'open'` to the class list... One advantage to this is that you can then use CSS transitions more easily.

(I don't *thing* CSS transitions work across the `style` attribute, at least)